pdp-info/info.go

```diff
@@ -0,0 +1,156 @@
+package info
+
+import (
+	"fmt"
+	"math/rand"
+	"strings"
+	"sync"
+	"time"
+
+	log "github.com/Sirupsen/logrus"
+)
+
+type streamObj struct {
+	stream PipService_GetRequestsStreamServer
+	err    chan error
+}
+
+type streamsPool struct {
```

Comment:
Could you add some comments to explain details how streamsPool work ?
1. how many streamsPools are needed for multiple PIP Server Vendors ?  For example: we have McAfee PIP server, we have Symantec PIP server, etc. 
2. After a streamsPool is initialized, if a PIP server or all PIP servers are restarted, how to re-create the stream connections and put them into the streamsPool ?

Reply:
We can use any count of PIP servers.
On PIP first connect PDP registers new stream with name that PIP returns.
If PIP is restarted - PDP removes broken stream and registers another one after PIP reconnect.